azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/FunctionDeployType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.appservice.model;
+
+import com.microsoft.azure.toolkit.lib.common.exception.AzureExecutionException;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+
+public enum FunctionDeployType {
+    FTP,
+    ZIP,
+    EMPTY,
```

Comment:
what is this used for?

Reply:
Deploy methods for function, like web app one deploy